pages/learn/studio/guides/scripting/server-values.md

```diff
@@ -0,0 +1,48 @@
+# **Server Values**
+
+## **Introduction**
+Server Values are server variables that send an event to the client when their value changes, enabling you to sync data between the client, server, and players.
+
+### **Value Types**
+
+`IntValue` `StringValue` `BoolValue` `Vector3Value` `NumberValue`
+
+#### Example:
+```lua
+local myInt = IntValue.new("MyInt", 0) -- Value.new(ServerIdentifier, Default Value)
+local myString = StringValue.new("MyString", "") -- Value.new(ServerIdentifier, Default Value)
+local myBool = BoolValue.new("MyBool", false) -- Value.new(ServerIdentifier, Default Value)
+```
+
+### **Changing Server Values**
+In the server you can set server values with the variable's `.value` property.
```

Comment:
Changing Values

On the server you can change the value using the `.value` property.

Reply:
Changing a value on the server will automatically broadcast that change to all clients.